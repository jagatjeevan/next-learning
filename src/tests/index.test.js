import renderer from 'react-test-renderer';
import Home from '../../pages/index';

describe('Home', () => {
  it('updates the snapshot', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
