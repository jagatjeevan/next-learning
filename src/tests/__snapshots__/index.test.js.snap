// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home updates the snapshot 1`] = `
<div
  className="container"
>
  <h1>
    Home page of the application
  </h1>
  Go to
  <a
    href="/about"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    About page
  </a>
  <button
    onClick={[Function]}
    type="button"
  >
    sdfgsdfgsdfg
  </button>
</div>
`;
